router/image/templates/ssl.conf

```diff
@@ -0,0 +1,25 @@
+# set the allowed protocols
+ssl_protocols {{ or (getv "/deis/router/sslProtocols") "TLSv1 TLSv1.1 TLSv1.2" }};
+
+# turn on session caching to drastically improve performance
+{{ if exists "/deis/router/sslSessionCache" }}
+ssl_session_cache {{ getv "/deis/router/sslSessionCache" }};
+ssl_session_timeout {{ or (getv "/deis/router/sslSessionTimeout") "10m" }};
+{{ end }}
+
+# turn off tickets, as they can cause issues in HA environments
+ssl_session_tickets off;
```

Comment:
I think this should be optional. The cost of not using session resumption is too high
https://blog.cloudflare.com/tls-session-resumption-full-speed-and-secure/